src/ddscxx/src/org/eclipse/cyclonedds/pub/qos/DataWriterQosDelegate.cpp

```diff
@@ -260,6 +271,7 @@ DataWriterQosDelegate::operator =(const org::eclipse::cyclonedds::topic::qos::To
     priority_    = tqos.policy<dds::core::policy::TransportPriority>();
     lifespan_    = tqos.policy<dds::core::policy::Lifespan>();
     ownership_   = tqos.policy<dds::core::policy::Ownership>();
+    datarepresentation_ = tqos.policy<dds::core::policy::DataRepresentation>();
```

Comment:
Please also encapsulate this line with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro.  (BTW: I see that in this particular case the embedding OMG_DDS_OWNERSHIP_SUPPORT macro was also missing for the "ownership" policy).

Reply:
Will do.